Makefile

```diff
@@ -107,6 +107,18 @@ start-postgres13:
 dev-postgres13: start-postgres13
 	cp $(CONFIG_PATH)/postgres13 $(CONFIG_FILE)
 
+.PHONY: start-pg-postgres13
+start-pg-postgres13: build-qe-napi build-connector-kit-js start-postgres13
+
+dev-pg-postgres13: start-pg-postgres13
+	cp $(CONFIG_PATH)/pg-postgres13 $(CONFIG_FILE)
+
+start-neon-postgres13: build-qe-napi build-connector-kit-js
```

Comment:
Should this one also be `.PHONY`?

Reply:
I think the other one doesn't need to be .PHONY, actually. Removing it.

Reply:
Good catch for consistency, BTW! Thank you!